Website/web/src/pages/public/AuthRegister.jsx

```diff
@@ -57,21 +59,30 @@ const AuthRegister = () => {
       <Formik
         innerRef={formRef}
         initialValues={{
+          first_name: "",
```

Comment:
This component is starting to get a little long. We should consider cleaning up the UI. Maybe making it a multistep registration form with a Stepper component?